projects/kit/components/tabs/tabs/tabs.component.ts

```diff
@@ -124,6 +128,20 @@ export class TuiTabsComponent implements AfterViewChecked {
 
         this.activeItemIndex = index;
```

Comment:
Please use the setter here

Reply:
why we need to use setter here? it will trigger 1 additional recalculation method. I think this line can be removed

Reply:
I don't get it. Why would it? It is programmatic setting after user. You mean it would bubble up and back with two way binding? What if user doesn't have binding at all and just uses router links?

Reply:
yeah, havent even thought about it :)